tools/server-spring/src/main/java/org/eclipse/rdf4j/http/server/repository/transaction/TransactionStartController.java

```diff
@@ -13,10 +13,7 @@
 import static javax.servlet.http.HttpServletResponse.SC_CREATED;
 
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.UUID;
+import java.util.*;
```

Comment:
Can you adjust your IDE settings to not replace imports with wildcards?

Reply:
done